apps/accounts/models/hosting.py

```diff
@@ -284,9 +286,26 @@ def shared_secret(self) -> str:
         except Hostingprovider.providersharedsecret.RelatedObjectDoesNotExist:
             return None
 
+    @property
     def public_supporting_evidence(self):
+        """
+        Return the supporting evidence that has explictly been marked as public
+        by the users uploading it to the database
+        """
         return self.supporting_documents.filter(public=True)
 
+    @property
+    def evidence_expiry_date(self) -> typing.List[datetime.date]:
```

Comment:
From the method body I assume the expected return type is `Optional[datetime.date]`